ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml

```diff
@@ -375,22 +451,17 @@ StackView {
                 text: qsTr("Delete")
                 type: StatusBaseButton.Type.Danger
 
-                visible: view.deployState === Constants.ContractTransactionStatus.Failed
+                visible: (!tokenViewPage.isPrivilegedTokenItem) & !root.isAdminOnly & tokenViewPage.deploymentFailed
```

Comment:
please don't use binary and operator `&` when the intention is to do logical and `&&`
```suggestion
                visible: !tokenViewPage.isPrivilegedTokenItem && !root.isAdminOnly && tokenViewPage.deploymentFailed
```

Reply:
Ohh! yes yes!! Fixed!